doc/connectivity/networking/index.rst

```diff
@@ -18,3 +18,4 @@ operation of the stacks and how they were implemented.
    network_tracing.rst
    api/index.rst
    conn_mgr/index.rst
+   dsa.rst
```

Comment:
The image is nice and it is ok to have it in png format. But if we ever needs to adjust the image in the future, could you also provide the original file in svg format?

Reply:
Sure. Will provide svg.